projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -518,15 +516,16 @@ export class DatagridComponent<R extends B, B = any> implements OnInit, AfterVie
      */
     @Output() selectionChanged = this.datagridSelectionChange;
 
+    private _actionDisplayConfig: DatagridActionDisplayConfig = getDefaultDatagridActionDisplayConfig();
     /**
      * How to display the static and contextual actions.
      */
-    @Input() actionDisplayConfig: ActionDisplayConfig;
-
-    /**
-     * Whether to display contextual actions inside the row or on top of the grid
-     */
-    @Input() contextualActionPosition: ContextualActionPosition = ContextualActionPosition.TOP;
+    @Input() set actionDisplayConfig(value: DatagridActionDisplayConfig) {
```

Comment:
Here it is good to denote what are the defaults, so if the developer does not  set it it will easily be seen.